### ISSUE TITLE
Blocklist: low contrast

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#85]
> The contrast between the small text and background is not sufficient. The contrast ratio is 2.38:1 when it should be 4.5:1 for text smaller than 18px.

<img width="103" height="81" alt="Image" src="https://github.com/user-attachments/assets/d158db63-cbf2-4b07-a31f-1bcaef9762b5" />

> The contrast between the small text and background is not sufficient. The contrast ratio is 3.8:1 when it should be 4.5:1 for text smaller than 18px.

<img width="85" height="50" alt="Image" src="https://github.com/user-attachments/assets/a672b2a9-a084-4d9e-a19b-c1a92cedbbb7" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
